Anagram-Task/src/AnagramOfName.java

```diff
@@ -0,0 +1,48 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class AnagramOfName {
+   /*     Анаграмма слова x - слово, по длине равное x и состоящее из таких же букв, что и x.
+          Например, слово “vani” является анаграммой слова “ivan”, а слова “naan” и “anan” являются анаграммами слова “anna”.
+          Дан словарь, содержащий анаграммы, например ;
+          public static String[] list = {“anna”, “ivan”, “naan”, “vani”, “piotr”, “nana”, “navi”}.
+          Написать функцию, которая по слову из словаря вернет все анаграммы этого слова, которые есть в словаре.
+          Например, по слову “ivan” функция вернет
+          "ivan" -> {"ivan", "navi", "vani"}
+    */
+
+    public static void main(String[] args) {
+        String name = "anna";
+        String[] anagrams = {"anna", "ivan", "naan", "vani", "piotr", "nana", "navi"};
+        System.out.println(createAnagramMap(anagrams, name));
+
+    }
+
+    public static String getSortedCharArrayFromString(String string) {
+        char[] charArr = string.toCharArray();
+        Arrays.sort(charArr);
+        return new String(charArr);
+    }
+
+    public static Map<String, List<String>> createAnagramMap(String[] anagrams, String name) {
```

Comment:
As it is specified in the task description, you should provide a method public `List<String> getAnagram(String word)`, which returns a list of anagrams for a given word. I do not see such method in your code, so, no solution. 

The `createAnagramMap()` supposes to create a map, which helps us to retrieve anagrams of a given word faster. It can be private since we do not expect clients to call it directly, it should probably be called only by a method in your class.

Reply:
I completed the task. Hope it's what was expected. Can you check it please.